corehq/apps/style/management/commands/fix_less_imports_collectstatic.py

```diff
@@ -23,11 +21,25 @@ def handle(self, *args, **options):
         for less_file in collectedless:
             with codecs.open(less_file, 'r', 'utf-8') as fd:
                 content = fd.read()
-                if NON_COLLECTED_PATH in content:
-                    print "[FIXING] bad less import in %s" % less_file
-                    content = content.replace(NON_COLLECTED_PATH, COLLECTED_PATH)
-                else:
-                    content = None
+                if content is not None:
+                    B3_REGEX = r"@\{b3-import-[a-z]+-[a-z]+\}"
+                    p = re.search(B3_REGEX, content)
+                    if p is not None:
+                        path_def = p.group(0)
+                        path_file = path_def.split('-')[-2]
+                        new_path = settings.LESS_B3_PATHS.get(path_file)
+                        print ("[CORRECTING] %(f_name)s path in %(less_file)s: "
```

Comment:
maybe move this to under the `new_path` if statement? not sure if it's a bad thing if it gets this far and `new_path` is `None`